corehq/apps/app_manager/static/app_manager/js/multimedia_size_util.js

```diff
@@ -0,0 +1,44 @@
+hqDefine('app_manager/js/multimedia_size_util',[
+    'jquery',
+    'underscore',
+    'knockout',
+    'hqwebapp/js/initial_page_data',
+], function ($, _, ko, initialPageData) {
+    var multimediaSize = function (name, size) {
+        var self = {};
+        self.name = ko.observable(name);
+        self.size = ko.observable(size);
+        return self;
+    }
+    var multimediaSizesView = function (url) {
+        var self = {};
+        self.sizes = ko.observableArray();
+        self.load_state = ko.observable(null);
```

Comment:
Nitpick: would be nice if casing matched, so `loadState` instead of `load_state`.

Reply:
https://github.com/dimagi/commcare-hq/pull/25864/commits/9528524cf0dbc8b66385b273c6c65ae188deed7d